docs/application/dotnet/guides/app-management/application-preference.md

```diff
@@ -0,0 +1,266 @@
+# Application Preferences
+
+You can manage application preferences by setting and getting them. You can also share stored preference data among applications in the same package.
```

Comment:
A little more intro on what is application preference? why a developed needs to set or get them will provide more value add.